.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+version: 2.1
+orbs:
+  node: circleci/node@3.0.1
```

Comment:
Is there a reason to not use the latest orb version at https://circleci.com/developer/orbs/orb/circleci/node (4.1.0)?